src/puppetlabs/trapperkeeper/services/webserver/jetty9_config.clj

```diff
@@ -318,13 +323,20 @@
     default-so-linger-in-milliseconds))
 
 (schema/defn ^:always-validate
+  common-connector-config :- WebserverConnectorCommon
+  [config :- WebserverRawConfig]
+  {:request-header-max-size   (or (:request-header-max-size config)
+                                  default-request-header-size)
+   :so-linger-milliseconds    (so-linger-in-milliseconds config)
+   :idle-timeout-milliseconds (:idle-timeout-milliseconds config)})
```

Comment:
I'm mostly in what you'd probably call the "other camp" on this one.

I totally agree that when exposing a new Jetty setting in tk-jetty9 configuration that we should draft from what Jetty is using as its internal default.  Unless we were to have a very solid reason to do otherwise, I agree that it's best to trust Jetty's judgement on what the reasonable starting point would be.

Where my opinion differs from others is around what should happen if the underlying default in Jetty were to change from one Jetty revision to another as we upgrade the tk-jetty9's Jetty dependency.  I fear that applications dependent upon the default value may be adversely affected by the default being changed on an upgrade.  For example, if core Jetty were to decide to change the `idle-timeout` from 30 seconds to 15 seconds in a later release and we were to let that new default flow through tk-jetty9 as-is, I could see the possibility for applications that have an implicit dependency on the 30 second behavior start having problems - e.g., more timed out connections - as the upgrade occurs.

Ideally such applications would isolate themselves from this risk by setting an explicit override for the settings where those applications have domain knowledge that would be more relevant than the Jetty default, e.g., an application serving requests in specific environments where it is known that the typical connection negotiation is right around the 30 second window.  I doubt it is all that common that those applications set such overrides out of the box, though.  I think it is more likely for such a setting to be updated when a problem occurs and getting to the understanding that updating the setting would fix a specific problem can be very time-consuming.

For the sake of consistency with how other settings have been layered in tk-jetty9 so far, I'd lean toward coding a default into tk-jetty9 for this PR - even though I could see this being changed in the future should the argument for trusting Jetty's defaults prevail.  Regardless of how this argument shakes out, I agree that our long-term goal should be for consistency in approach in how we layer these settings across the service implementation, even if getting to that involves work beyond the scope of this PR.

If the majority is in favor of deferring to Jetty's default for this PR, I think @KevinCorcoran raises a good point about how we want to reflect Jetty's default in external documentation.  I really like having documentation which states explicitly what the application's default is -- if nothing else than for understanding in the context of troubleshooting.  If we do document a default provided by Jetty, however, I think we need to be clear that in doing so we:

1) Are not committing to that default being preserved across upgrades.

2) Are accurately reflecting the value that Jetty is using.

Pertaining to the second point, I'd be concerned that if we don't have any tests in place to validate what Jetty is using as a default that we'd run the risk of upgrading the tk-jetty9 dependency on Jetty, not noticing that the effective default had changed, and, therefore, not updating the documentation to have the correct value.

If we decide to go with letting Jetty's default flow through for this PR, I think the simplest thing to do would be to remove the documentation about what the default is, per @KevinCorcoran's suggestion.